lib/plugins/aws/provider.js

```diff
@@ -1973,7 +1973,7 @@ Object.defineProperties(
           };
         }
 
-        const [repositoryName, imageTag] = image.split('/')[1].split(':');
+        const [repositoryName, imageTag] = image.split('/').slice(1).join('/').split(':');
```

Comment:
Maybe let's use something as: `image.slice(image.indexOf('/') + 1).split(':')` (feels more readable to me)

Reply:
Changed :+1: